contrib/install.sh

```diff
@@ -25,7 +25,7 @@ for SRC in $ARGS; do
 
     # Do the chmod dance, and ignore errors on platforms that don't like setting permissions of symlinks
     # TODO: Test if it's a symlink instead of having to redirect stderr to /dev/null
-    chmod $PERMS $DESTFILE 2>/dev/null
+    chmod $PERMS $DESTFILE 2>/dev/null || exit 1
```

Comment:
Yeah I remember seeing the comment above the line. I did this for my testing purposes so that I would not miss any potential problems. I will revert this.